features/admin.registration-flow-builder.v1/constants/registration-flow-ai-constants.ts

```diff
@@ -46,13 +46,13 @@ export const useGetStatusLabels = (): Record<string, string> => {
 };
 
 export const STATUS_PROGRESS_MAP: Record<string, number> = {
```

Comment:
Will this come from an API from the AI service in the future?
Also should we use camelcase or snakecase here?